kale/embed/decomposition.py

```diff
@@ -258,3 +263,144 @@ def inverse_transform(self, x):
         x_rec = x_rec + self.mean_
 
         return x_rec
+
+
+class MIDA(BaseEstimator, TransformerMixin):
```

Comment:
Is there a test for this new calss? I did not find any. It will be good to have one to maintain our coverage at 93%. Thanks.